app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/mnemonics/RestoreMnemonicsScreen.kt

```diff
@@ -320,7 +320,7 @@ private fun EntitiesView(
                     )
                 }
 
-                if (state.isMainBabylonSeedPhrase) {
+                if (state.isMainBabylonSeedPhrase && state.isMandatory.not()) {
```

Comment:
Why do we need this ?

Reply:
becuase skip button is visible and you can skip mandatory restore of BDFS, see point 4b